README.md

```diff
@@ -82,7 +84,13 @@ For a future use you can omit the second step and run `up` (the third step) dire
 ### Windows
 
 Currently not supported, but should work using docker as well.
+### Bitcoin regtest
+Optionally you can run bitcoin regtest backend with trezor-user-env. This will run another docker container that contains bitcoind in regtest mode and our blocbook backend connected to that bitcoind in regtest mode.
```

Comment:
Typo - `blocbook`

Reply:
On the optionallity - `controller` is currently always connecting to the regtest at the beginning, so we should have error handling in place when the regtest would not be running